controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +613,42 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands: 2.4,5H,5L,60.
```

Comment:
I misled you after I misled myself,
the value should be 6 and not 60. please revert back the change

Reply:
Changed